pkg/controller/registry/reconciler/reconciler_test.go

```diff
@@ -1,7 +1,10 @@
 package reconciler
 
 import (
+	"fmt"
```

Comment:
nit: imports

Reply:
imports from the std-lib should be grouped together

Reply:
is there a make command or something to check that? this is how VSCode automatically imported it for me

Reply:
there is a [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) command that will sort the imports. I'm not sure if it will break them out into groups correctly, but that's something we like to do in our repos -- it makes external dependencies easier to see. 

Per @joelanford we can breakup imports into three groups:
* standard library packages
* other packages from within the project itself
* external dependencies

Reply:
let me know if https://github.com/operator-framework/operator-lifecycle-manager/pull/2023/commits/fe24a8ad57434ee19341291c30da53852320fd1a has the correct grouping